src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -54,10 +53,16 @@ private Reflector() { }
   public static void setField(
       final Object obj,
       final String fieldName,
-      final Object newVal)
-      throws
-      NoSuchFieldException, IllegalArgumentException, IllegalAccessException {
-    setField(obj.getClass(), obj, fieldName, newVal);
```

Comment:
Should this be replaced with `throws DeeptestUtilsException`? The same would apply to all the other methods below.

Reply:
As it stands, DeeptestUtilsException is a RuntimeException, so it would not be in the `throws` list (which is exactly why I did it, so it doesn't have to be printed in the test method either). Whether it should be a RuntimeException or not is a question posed by Thomas. Do you have any opinion/advice on that?